hivemind/p2p/p2p_daemon_bindings/datastructures.py

```diff
@@ -131,6 +131,9 @@ def from_protobuf(cls, peer_info_pb: p2pd_pb2.PeerInfo) -> "PeerInfo":
     def __str__(self):
         return f"{self.peer_id.pretty()} {','.join(str(a) for a in self.addrs)}"
 
+    def __repr__(self):
+        return str(self)
```

Comment:
```suggestion
        return f"PeerInfo(peer_id={repr(self.peer_id)}, addrs={repr(self.addrs)})"
```

`repr()` usually shows the class name and contents in a Python-like notation.